src/rpdk/core/contract/suite/handler_create.py

```diff
@@ -51,14 +51,14 @@ def contract_create_delete(resource_client):
 
 @pytest.mark.create
 @skip_not_writable_identifier
-def contract_create_duplicate(created_resource, resource_client):
+def contract_create_create(created_resource, resource_client):
```

Comment:
Gentle reminder- Update any reference to old names in documentation, if not done already.